diktat-rules/src/test/resources/test/smoke/src/main/kotlin/Bug1Expected.kt

```diff
@@ -1,19 +1,19 @@
-// ;warn:1:1: [FILE_NAME_MATCH_CLASS] file name is incorrect - it should match with the class described in it if there is the only one class declared: Bug1Expected.kt vs D{{.*}}
+// ;warn:$line:1: [FILE_NAME_MATCH_CLASS] file name is incorrect - it should match with the class described in it if there is the only one class declared: Bug1Expected.kt vs D{{.*}}
```

Comment:
I remember - we were adding functionality for NO WRITING A LINE NUMBER in warning. 
Does it work?

Reply:
We cannot completely remove here from the line number. Because Diktat does not know how to correctly define the warning line.

Reply:
why diktat? I am saying about save

Reply:
Fixed all